### PR TITLE
CI - First version for macOS

### PR DESCRIPTION
**Pull request by yoav-lavi**
# Description

## Tooling

- Sets up a CI to build and sign universal macOS `.dmg`s

# Type of change

- [ ] 💔 Breaking
- [ ] 🚀 Feature
- [ ] 🐛 Fix
- [x] 🛠️ Tooling
- [ ] 🔨 Refactoring
- [ ] 🧪 Test
- [ ] 📦 Dependency
- [ ] 📖 Requires documentation update

### CODE REVIEW

**Comment by yoav-lavi** [comment]
May change later on to tags, the version must be incremented for this to work

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,166 @@
+name: "publish"
```
Comment (fbjork) [inline .github/workflows/ci.yml:1]:
```suggestion
name: CI
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    outputs:
+      release_id: ${{ steps.create-release.outputs.result }}
+
+    steps:
+      - uses: actions/checkout@v3
+      - run: |
+          PACKAGE_JSON_PATH="./apps/desktop"
+          echo "PACKAGE_VERSION=$(cat ${PACKAGE_JSON_PATH}/package.json | jq ".version" | tr -d '""')" >> $GITHUB_ENV
+        shell: bash
+      - name: Setup node
```
Comment (fbjork) [inline .github/workflows/ci.yml:22]:
```suggestion
      - name: Setup Node
```

Comment (yoav-lavi) [reply .github/workflows/ci.yml:22]: Done

File: .github/workflows/ci.yml (hunk 1 above)
Comment (yoav-lavi) [reply .github/workflows/ci.yml]: Done

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,167 @@
+name: "CI"
+
+on:
+  push:
+    branches:
+      - main
```
Comment (obmarg) [inline .github/workflows/ci.yml:6]: So this action looks like it tries to create a new release w/ the version from package.json - but we're running it on every push to main?  Seems like it's going to cause clashes pretty easily.

**Review by obmarg — APPROVED** [review]

**Pull request merged**